src/surge-xt/gui/SurgeGUIEditorHtmlGenerators.cpp

```diff
@@ -446,8 +446,10 @@ struct oscParamInfo
 // Sort function for displaying parameters (below)
 bool compareParams(const oscParamInfo &opl, const oscParamInfo &opr)
 {
-    int lcg = opl.p->ctrlgroup;
-    int rcg = opr.p->ctrlgroup;
+    int cgroup_order[10] = {0, 1, 2, 4, 3, 5, 6, 7, 8, 9};
+
+    int lcg = cgroup_order[opl.p->ctrlgroup];
+    int rcg = cgroup_order[opr.p->ctrlgroup];
```

Comment:
I don't understand the necessity of cgroup_order here. It looks like it's mapping 0-9 to 0-9.

Reply:
Four and three are transposed. It was just a way to arrange the control groups on the HTML with a better flow.